tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/openwire/SimpleOpenWireTest.java

```diff
@@ -964,7 +964,7 @@ public void testAutoDestinationCreationOnProducerSend() throws JMSException {
 
       MessageConsumer consumer = session.createConsumer(queue);
       TextMessage message1 = (TextMessage) consumer.receive(1000);
-      assertTrue(message1.getText().equals(message.getText()));
+      assertEquals(message1.getText(), message.getText());
```

Comment:
Args are reversed.

(The message.getText() would also be better replaced as a variable, also passed instead of a literal in the earlier `session.createTextMessage` )